Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM fedora
+
+COPY . ./BUILD/
```

Comment:
Please don't include sources in the container. This requires rebuilding it every time you change any source file.